src/test/java/me/gnat008/perworldinventory/commands/SetWorldDefaultCommandTest.java

```diff
@@ -34,6 +35,8 @@
 /**
  * Tests for {@link SetWorldDefaultCommand}.
  */
+// TODO Gnat008: Fix tests so they work again
```

Comment:
I think all the tests for this one need to be reworked, and I'm struggling with how to do it currently.